fixture.js

```diff
@@ -16,3 +16,14 @@ var cli = meow({
 Object.keys(cli.flags).forEach(function (el) {
 	console.log(el);
 });
+
+cli.stdinOrInput()
```

Comment:
Maybe `cli.autoInput()` or `cli.getInput()` or something along those lines would look better and shorter?